q-and-a.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: Q&A
+permalink: /q-and-a/
+---
+
+# Q&A
+
+Op deze pagina hebben we de antwoorden verzameld op vragen die we regelmatig tegenkomen. Mis je een vraag? Of wil je ergens over doorpraten? [Stuur ons dan een bericht](/contact).
+
+## Wat is Nuts?
+
+Nuts is een samenwerkingsverband dat ervoor wil zorgen dat zorgverleners digitaal kunnen samenwerken. Als basis hanteren we een set uitgangspunten, die we hebben gevat in een [manifest](/manifest). Iedere partij die zich kan vinden in deze uitgangspunten, kan zich aansluiten bij Nuts. Op basis van deze uitgangspunten verzamelen en bouwen we de [technische componenten](/componenten) die nodig zijn.
+
+## Hoe is Nuts ontstaan?
+
+Nuts is ontstaan vanuit een gedeelde realisatie onder software leveranciers: we zijn veel tijd kwijt met talloze maatwerk koppelingen te bouwen tussen elkaars systemen. Deze koppelingen zijn meestal kostbaar, tijdrovend en foutgevoelig. Hierdoor is het vandaag de dag voor veel zorgverleners niet goed mogelijk om digitaal samen te werken met collega's die gebruik maken van een ander ICT-systeem. In 2018 hebben deze leveranciers de handen in een geslagen en zijn het Nuts initiatief gestart. Dit geen geen praatgroepje maar een bouwgroep!
```

Comment:
i.p.v. Dit is geen praatgroepje...

Essentieel is dat we het op zo'n manier organiseren, dat we ook daadwerkelijk de beoogde technologie gerealiseerd krijgen.